### ISSUE TITLE
Is it possible to define lags_past_covariates in PastCovariatesTorchModel?

### ISSUE THREAD

**Issue by Pachark** [unit8co/darts#1664]
When using PastCovariatesTorchModel such as BlockRNNModel, NBEATSModel, is it possible to define lags_past_covariates? 

I have a multivariate dataset that contains 1 target and 9 covariates. 
If I set input_chunk_length as 24, the model will accept 24 lags * (1+9) = 240 dimension input. I think the model design may easily overfit because the size of my dataset is small. So I only want to accept 24 lags of target, and 1 lag of covariates (T-1 only). 

If I understand correctly, the lags_past_covariates in LightGBMModel perform exactly the same as the above scenario.
Is it possible to concatenate the target with 1 lag of covariates and form a fully-connected layer in PastCovariatesTorchModel ?

**Comment by dennisbader** [comment]
No, this is currently not possible. It might be something we could add in the future though.
